src/config/format.ts

```diff
@@ -0,0 +1,27 @@
+export type formatKeysType = {
+  VALUE: '$value' | 'value',
```

Comment:
Do you plan to write (or know of another project) a new Format for style dictionary that will recognize `$value` ?

Reply:
Yes, currently you need a custom parser as style dictionary does not work if there is no `value`.

I will add the parser with this PR to the examples.